examples/node14/.lando.yml

```diff
@@ -0,0 +1,40 @@
+name: lando-node-14
+services:
+  defaults:
+    type: node:14
+    build:
+      - yarn
+    command: /app/node_modules/.bin/nodemon src/app-http.js --watch src --ignore *.test.js
+  cli:
+    type: node
+  compass:
+    type: node:14
+    globals:
+      grunt-cli: latest
+    build_as_root:
+      - apt-get update -y
+      - apt-get install ruby-full -y
+      - gem install compass
+  custom:
+    type: node:16
+    ssl: true
+    globals:
+      gulp-cli: latest
+    port: 3000
+    build:
+      - yarn
+    command: /app/node_modules/.bin/nodemon src/app-https.js --watch src --ignore *.test.js
+  custom2:
+    type: node:14.18
+    ssl: 4444
+    port: 3000
+    build:
+      - yarn
+    command: /app/node_modules/.bin/nodemon src/app-custom.js --watch src --ignore *.test.js
+  patch:
+    type: node:14.18.3
```

Comment:
I don't think this actually exists.

Reply:
Ouch, you're quite right about that, should be fixed now.